Form/Type/AdminType.php

```diff
@@ -67,11 +67,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                             $subject = $subjectCollection->get(trim($options['property_path'], '[]'));
                         }
                     } else {
+                        // this check is to work around duplication issue in property path
+                        // https://github.com/sonata-project/SonataAdminBundle/issues/4425
+                        if ($this->getFieldDescription($options)->getFieldName() === $options['property_path']) {
+                            $path = $options['property_path'];
+                        } else {
+                            $path = $this->getFieldDescription($options)->getFieldName().$options['property_path'];
```

Comment:
When does this happen and what does it typically produce? Is there no need for a separator here? Is the left part the parent name, and the right part the child name?

Reply:
I would like to know if this is related to using some concrete version of other bundle and can be removed on a next major version like 4.0 of SonataAdmin, or this will stay like this forever